src/main/java/de/tum/in/www1/artemis/web/rest/StudentExamResource.java

```diff
@@ -488,14 +489,7 @@ private void fetchParticipationsSubmissionsAndResultsForStudentExam(StudentExam
         // connect & filter the exercises and student participations including the latest submission and results where necessary, to make sure all relevant associations are
         // available
         for (Exercise exercise : studentExam.getExercises()) {
-            if (forConduction) {
```

Comment:
`forConduction` needs to be removed from the function parameters, as it is not used anymore.